tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py

```diff
@@ -21,6 +21,7 @@
     skipif_ocp_version,
     performance,
     performance_b,
+    ignore_leftovers,
```

Comment:
same as above, will be good to handle the clone failure.

Reply:
Yes, the snapshots are deleted in the teardown (lines 123-129)